frontend/www/js/omegaup/components/homepage/Sponsors.vue

```diff
@@ -1,12 +1,10 @@
 <template>
   <div class="row p-4 text-center justify-content-around align-items-center">
     <h3 class="col-12 mb-3 display-4">{{ title }}</h3>
-    <div
-      v-for="(logoSrc, index) in logosSrc"
-      :key="index"
-      class="p-3 mt-2 mt-md-0"
-    >
-      <img class="img-fluid" :src="logoSrc" />
+    <div v-for="(logo, index) in logos" :key="index" class="p-3 mt-2 mt-md-0">
```

Comment:
```suggestion
    <div v-for="(logo, index) in logos" :key="logo.href" class="p-3 mt-2 mt-md-0">
```

en los `key` nunca deben ir índices: si se cambia el arreglo, Vue no tiene manera de saber que hay que repintar ese elemento porque el índice no cambia!

Reply:
oh, gracias!